### PR TITLE
Add awscli and aws-azure-login to standard

### PR DESCRIPTION
**Pull request by yuku**
## What
* Add [awscli](https://aws.amazon.com/cli/) and [aws-azure-login](https://github.com/sportradar/aws-azure-login).
* Use `brew bundle --no-lock` option to suppress outputting Brewfile.macos.lock.json file.

## Why
* Many repositories depend on them.
* Currently, we don't find any problem without sharing the Brewfile.macos.lock.json file.
    * If we actually need to lock tools' versions, we will remove the `--no-lock` option and commit the file.

## Ref
* https://aws.amazon.com/cli/
* https://github.com/sportradar/aws-azure-login

### CODE REVIEW

File: bin/setup [hunk 1]
```diff
@@ -62,6 +62,24 @@ def inject(text: str, check: str, to: str) -> None:
             f.write(dedented)
             print(f"Write to {to}:\n{indented}")
 
+def setup_aws_azure_login(path: str = "/usr/local/bin/aws-azure-login") -> None:
```
Comment (yuku) [inline bin/setup:65]: Use Docker to run aws-azure-login to make sure that the command is always available:

* Our standard knows Docker and nodenv exist, but it doesn't guarantee there is an active nodejs env.
* In addition, if we install the command by `npm install -g aws-azure-login`, the command is going to be installed to the current environment. It means switching nodenv hides the command.

File: docs/local_environment.md [hunk 2]
```diff
@@ -3,19 +3,21 @@
...
+* [bash](https://www.gnu.org/software/bash/)
+* [Docker](https://www.docker.com/)
+* [git](https://git-scm.com/)
+* [Homebrew](https://brew.sh)
+* [jsonnet](https://jsonnet.org/)
+* [kubectl](https://kubernetes.io/docs/reference/kubectl/kubectl/)
+* [nodenv](https://github.com/nodenv/nodenv)
+* [Pipenv](https://pipenv.readthedocs.io/en/latest/)
+* [pyenv](https://github.com/pyenv/pyenv)
+* [Yarn](https://yarnpkg.com/)
```
Comment (yuku) [inline docs/local_environment.md:18]: * Add awscli and aws-azure-login then sort by dictionary order.
* Remove 🚀 to reduce redundancy.

File: bin/setup [hunk 3]
```diff
@@ -62,6 +62,24 @@ def inject(text: str, check: str, to: str) -> None:
...
+
+    https://github.com/sportradar/aws-azure-login#docker
+    """
+    if not shutil.which("aws-azure-login"):
+        execute(
+            command=f"sudo curl -o {path} https://raw.githubusercontent.com/sportradar/aws-azure-login/main/docker-launch.sh",
+            prologue="Installing aws-azure-login",
+            error_message="Failed to install aws-azure-login",
+        )
+    if not os.access(path, os.X_OK):
```
Comment (shinyafw) [inline bin/setup:76]: nit: Do we need to check if the `path` exists?

```python
if os.path.exists(path) and not os.access(path, os.X_OK):
```

Comment (yuku) [reply bin/setup:76]: good catch 👍

**Pull request merged**